TechAndCheck/zenodotus
# Add media detail page

This PR adds a "show" page for individual archive items:

![media-show](https://user-images.githubusercontent.com/4731/202240553-eb36ed9f-e4af-475b-9a52-7a6879f18ac5.jpeg)

**Testing:**
1. `./bin/dev`
2. Go to the Dashboard
3. Click on the "Archived at…" timestamp metadata of any item
4. ✅ Find yourself on the media page for that item
5. Go back to the Dashboard
6. Click on the image itself for any non-video image
7. ✅ Find yourself again on the media page for that item

Note that we only link the image and the "Archived at…" timestamp (and label), **not** the caption or non-image (aka video) media, because those have other actions already.

Closes #366
Closes #405
